vcf_merge.wdl

```diff
@@ -6,12 +6,16 @@ task xVCFMerge {
         String output_file
         String billing_project
         String workspace
+        Int? addl_disk_space = 1
         Int? cpu = 8
         Int? memory = 64
         Int? preemptible = 0
     }
+    Int input_size = ceil(size(input_files, "GB"))
```

Comment:
Doesn't size return the file size in bytes?  Which means you're only adding 1 byte to the total size below?  Is that intended?

Reply:
By default, size() does indeed return the file size in bytes. But because I specified "GB" it will actually return the size in GB. See the spec here: https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#float-sizefile-string

Additionally, Cromwell's disks runtime attribute expects the disk size in GB too: https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#disks

(No, I don't know why the default behavior of size() gives a unit that is 10*-9 times smaller than the default unit that the thing that every really uses size() expects.)

Reply:
Side note: "G" and "GB" are both acceptable for this unit; the memory runtime attribute which just uses "G" is also valid. I just think "GB" is a little more human readable.